strax/storage/files.py

```diff
@@ -281,14 +281,20 @@ def _chunk_filename(self, chunk_info):
 
     def _save_chunk(self, data, chunk_info, executor=None):
         filename = self._chunk_filename(chunk_info)
-
         fn = os.path.join(self.tempdirname, filename)
-        kwargs = dict(data=data, compressor=self.md['compressor'])
+        
+        for compressor in strax.to_str_tuple(self.md['compressor']):
```

Comment:
Where does this quantity get set? It might be prudent to use `get('compressor', some_default_value)` instead. Also, what type are we expecting this to take? A specific compressor? A list of desired compressors? I thought the initial idea was to default to blosc unless it's too large, in which case switch to lz4.

Reply:
Sorry, somehow i missed the activity on this thread. As you can see the current implementation assumes there is a compressor set for each plugin. The default value is already set to "blosc" since the compressor property of the Plugin base class is "blosc".  As I mentioned in my initial comment, the type is a list of compression names or a single name. It will use the first one in the list that can be used. if none of the compressions in the list can be used then it will throw an error. If you change the compression property of the Plugin class to ["blosc", "lz4"] you would get the desired behavior you want.